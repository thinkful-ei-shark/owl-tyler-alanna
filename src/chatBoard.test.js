import React from 'react';
import ReactDOM from 'react-dom';
import renderer from 'react-test-renderer';
import ChatBoard from './chatBoard';

it ('renders without crashing', ()=> {
    const div = document.createElement('div');

    ReactDOM.render(<ChatBoard />, div);

    ReactDOM.unmountComponentAtNode(div);
});

it('renders the UI as expected', () => {
    const tree = renderer
      .create(<ChatBoard/>)
      .toJSON();
    expect(tree).toMatchSnapshot();  
    });