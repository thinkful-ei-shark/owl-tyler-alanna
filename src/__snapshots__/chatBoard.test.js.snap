// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders the UI as expected 1`] = `
<div
  className="chatBoard"
>
  <div
    className="message"
  >
    <div>
      <img
        alt=""
      />
      <h3>
        Axe Kubicka
      </h3>
      <p
        className="timestamp"
      >
        1548852484247
      </p>
    </div>
    <div>
      <p>
        Hello world
      </p>
    </div>
  </div>
  <div
    className="event"
  >
    <h3>
      Caty Flucker
    </h3>
    <p>
      gave a thumbs up!
    </p>
    <p
      className="timestamp"
    >
      1548852484247
    </p>
  </div>
  <div
    className="event"
  >
    <h3>
      Frank Runciman
    </h3>
    <p>
      gave a thumbs down!
    </p>
    <p
      className="timestamp"
    >
      1548852484247
    </p>
  </div>
  <div
    className="event"
  >
    <h3>
      Koren Templeton
    </h3>
    <p>
      raised their hand!
    </p>
    <p
      className="timestamp"
    >
      1548852544247
    </p>
  </div>
  <div
    className="event"
  >
    <h3>
      Koren Templeton
    </h3>
    <p>
      clapped!
    </p>
    <p
      className="timestamp"
    >
      1548852544247
    </p>
  </div>
  <div
    className="event"
  >
    <h3>
      Koren Templeton
    </h3>
    <p>
      joined this session!
    </p>
    <p
      className="timestamp"
    >
      1548852544247
    </p>
  </div>
  <div
    className="event"
  >
    <h3>
      Ashla Attwool
    </h3>
    <p>
      left this session!
    </p>
    <p
      className="timestamp"
    >
      1548852604247
    </p>
  </div>
  <div
    className="event"
  >
    <h3>
      Axe Kubicka
    </h3>
    <p>
      joined the stage!
    </p>
    <p
      className="timestamp"
    >
      1548852664247
    </p>
  </div>
  <div
    className="event"
  >
    <h3>
      Axe Kubicka
    </h3>
    <p>
      left the stage!
    </p>
    <p
      className="timestamp"
    >
      1548852724247
    </p>
  </div>
</div>
`;
